crates/biome_js_formatter/src/js/module/export_named_clause.rs

```diff
@@ -31,12 +31,13 @@ impl FormatNodeRule<JsExportNamedClause> for FormatJsExportNamedClause {
                 [format_dangling_comments(node.syntax()).with_block_indent()]
             )?;
         } else {
+            let should_insert_space_around_brackets = f.options().bracket_spacing().value();
             write!(
                 f,
-                [group(&format_args![
-                    soft_line_indent_or_space(&specifiers.format()),
-                    soft_line_break_or_space(),
-                ])]
+                [group(&soft_block_indent_with_conditional_space(
+                    &specifiers.format(),
+                    should_insert_space_around_brackets
+                ),)]
```

Comment:
There are a few places like this where I changed the actual logic going on, but I believe they end up being syntactically equivalent. This change didn't break any existing tests, and logically it feels like the same result as well.